### ISSUE TITLE
PnP search filter is not accessible with tab key

### ISSUE THREAD

**Issue by MariaAug** [microsoft-search/pnp-modern-search#1296]
**Version used**
4.2.2.0

**Describe the bug**
PnP search filter is not accessible when using tab key.

**To Reproduce**
1. Add PnP search filter web part and PnP search box web part on SharePoint page.
2. Save the page
3. Start pressing tab key from the start of the page using NVDA speech viewer tool
4. The focus does not go to the filter web part and the tool does not read that there is a filter option available for the user.

**Expected behavior**
 Tab focus must be provided for the interactive elements on the search web parts and the same should be accessible by tab key.

**Screenshots**
![image](https://user-images.githubusercontent.com/90712499/133300311-02a299b7-073a-4a44-bddf-a6a9032ec35d.png)

**Comment by wobba** [comment]
Adding proper roles and tab index is hard work :)

**Comment by CollinHunterGOC** [comment]
Not working in 4.7.0.0

**Comment by wobba** [comment]
@CollinHunterGOC can you log a new issue for the latest version and describe what is missing from the markup to make this work?

**Comment by CollinHunterGOC** [comment]
Sure.
